ASIM/schemas/entities/ASimActor.yaml

```diff
@@ -62,3 +62,15 @@ Fields:
   Class: Optional
   Description: The unique ID of the sign-in session of the Actor.
   Example: 102pTUgC3p8RIqHvzxLCHnFlg
+
+- Name: ActorDomain
+  Type: string
+  Class: Recommended
+  Description: The actor's domain.
+
+- Name: ActorDomainType
```

Comment:
can we do TargetDomain? do we have similar already?

Reply:
Yes we have it in ASimUser.yaml